### DISCUSSION TITLE
🐛 | web interface loads very slow when mounting folders via fstab

### DISCUSSION THREAD

**Discussion by TrustCompact** [MiczFlor/RPi-Jukebox-RFID#2328] [Q&A]
## Bug
The web interface takes huge amount of time for loading. I'm talking about intervals greater than 80 seconds.
When loaded, usability is ok on the specific site. Not fast but ok.
On every new page-load, loading-time is persistent.

Everything else works fine.
Ping of IP works as intended and in normal time.

### What I did
I installed the raspberry pi with above mentioned buster image and ran the installer script

### I expected this to happen
Normal loading times

### Further information that might help
pi@Phoniebox:~ $ tail -n 500 /var/log/syslog
Apr 14 15:34:48 Phoniebox rsyslogd:  [origin software="rsyslogd" swVersion="8.1901.0"x-pid="386" x-info="https://www.rsyslog.com"] rsyslogd was HUPed
Apr 14 15:34:48 Phoniebox systemd[1]: phpsessionclean.service: Succeeded.
Apr 14 15:34:48 Phoniebox systemd[1]: Started Clean php session files.
Apr 14 15:34:49 Phoniebox rsyslogd:  [origin software="rsyslogd" swVersion="8.1901.0"x-pid="386" x-info="https://www.rsyslog.com"] rsyslogd was HUPed
Apr 14 15:34:49 Phoniebox systemd[1]: logrotate.service: Succeeded.
Apr 14 15:34:49 Phoniebox systemd[1]: Started Rotate log files.
Apr 14 15:34:50 Phoniebox systemd[1]: man-db.service: Succeeded.
Apr 14 15:34:50 Phoniebox systemd[1]: Started Daily man-db regeneration.
Apr 14 15:34:51 Phoniebox systemd[1]: apt-daily.service: Succeeded.
Apr 14 15:34:51 Phoniebox systemd[1]: Started Daily apt download activities.
Apr 14 15:34:51 Phoniebox systemd[1]: Starting Daily apt upgrade and clean activities...
Apr 14 15:34:52 Phoniebox gpio-buttons.py[490]: <xxxxxxxxxx>
Apr 14 15:34:52 Phoniebox gpio-buttons.py[490]: [paused]  #1/2   0:38/42:15 (1%)
Apr 14 15:34:52 Phoniebox gpio-buttons.py[490]: volume: 13%   repeat: off   random: off   single: off   consume: off
Apr 14 15:34:53 Phoniebox systemd[1]: apt-daily-upgrade.service: Succeeded.
Apr 14 15:34:53 Phoniebox systemd[1]: Started Daily apt upgrade and clean activities.
Apr 14 15:35:21 Phoniebox systemd[1]: phoniebox-idle-watchdog.service: Succeeded.
Apr 14 15:39:01 Phoniebox systemd[1]: Starting Clean php session files...
Apr 14 15:39:01 Phoniebox CRON[1138]: (root) CMD (  [ -x /usr/lib/php/sessionclean ]&& if [ ! -d /run/systemd/system ]; then /usr/lib/php/sessionclean; fi)
Apr 14 15:39:02 Phoniebox systemd[1]: phpsessionclean.service: Succeeded.
Apr 14 15:39:02 Phoniebox systemd[1]: Started Clean php session files.
Apr 14 15:39:30 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:40:08 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:40:16 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:40:53 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:41:10 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:41:30 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:42:07 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:43:07 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:43:14 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:44:05 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:44:53 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:45:10 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:46:12 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:46:20 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:46:58 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:47:04 Phoniebox systemd[1]: Created slice User Slice of UID 1000.
Apr 14 15:47:04 Phoniebox systemd[1]: Starting User Runtime Directory /run/user/1000...
Apr 14 15:47:04 Phoniebox systemd[1]: Started User Runtime Directory /run/user/1000.
Apr 14 15:47:04 Phoniebox systemd[1]: Starting User Manager for UID 1000...
Apr 14 15:47:05 Phoniebox systemd[2231]: Reached target Paths.
Apr 14 15:47:05 Phoniebox systemd[2231]: Listening on GnuPG cryptographic agent and passphrase cache.
Apr 14 15:47:05 Phoniebox systemd[2231]: Reached target Timers.
Apr 14 15:47:05 Phoniebox systemd[2231]: Listening on GnuPG cryptographic agent and passphrase cache (access for web browsers).
Apr 14 15:47:05 Phoniebox systemd[2231]: Listening on GnuPG cryptographic agent and passphrase cache (restricted).
Apr 14 15:47:05 Phoniebox systemd[2231]: Listening on GnuPG network certificate management daemon.
Apr 14 15:47:05 Phoniebox systemd[2231]: Listening on GnuPG cryptographic agent (ssh-agent emulation).
Apr 14 15:47:05 Phoniebox systemd[2231]: Reached target Sockets.
Apr 14 15:47:05 Phoniebox systemd[2231]: Reached target Basic System.
Apr 14 15:47:05 Phoniebox systemd[1]: Started User Manager for UID 1000.
Apr 14 15:47:05 Phoniebox systemd[2231]: Starting Music Player Daemon...
Apr 14 15:47:05 Phoniebox systemd[1]: Started Session c1 of user pi.
Apr 14 15:47:06 Phoniebox mpd[2244]: exception: failed to open log file "/var/log/mpd/mpd.log" (config line 40): Permission denied
Apr 14 15:47:06 Phoniebox systemd[2231]: mpd.service: Main process exited, code=exited, status=1/FAILURE
Apr 14 15:47:06 Phoniebox systemd[2231]: mpd.service: Failed with result 'exit-code'.
Apr 14 15:47:06 Phoniebox systemd[2231]: Failed to start Music Player Daemon.
Apr 14 15:47:06 Phoniebox systemd[2231]: Reached target Default.
Apr 14 15:47:06 Phoniebox systemd[2231]: Startup finished in 1.462s.
Apr 14 15:47:07 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:47:20 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:47:52 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:48:09 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:48:24 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:48:46 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:49:12 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:49:12 Phoniebox systemd[1]: Starting Cleanup of Temporary Directories...
Apr 14 15:49:12 Phoniebox systemd[1]: systemd-tmpfiles-clean.service: Succeeded.
Apr 14 15:49:12 Phoniebox systemd[1]: Started Cleanup of Temporary Directories.
Apr 14 15:49:49 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:50:03 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:50:15 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:51:09 Phoniebox lighttpd[866]: echo: write error: Broken pipe
Apr 14 15:51:20 Phoniebox lighttpd[866]: echo: write error: Broken pipe

### Base image and version
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

### Branch / Release
Refresh index: 100% (309/309), done.
On branch master
Your branch is up to date with 'origin/master'.

### Installscript
i.e. `scripts/installscripts/buster-install-default.sh`

### RaspberryPi version
3 B+

### RFID Reader
Van Ooijen Technische Informatica Keyboard

### Soundcard
C-Media Electronics, Inc. CM108 Audio Controller

### Other notable hardware

GPIO Buttons:
    volU = Button(22,pull_up=True,hold_time=0.3,hold_repeat=True)
    volD = Button(27,pull_up=True,hold_time=0.3,hold_repeat=True)
    frwd = Button(17,pull_up=True)
    prev = Button(24,pull_up=True)
    halt = Button(23,pull_up=True)

**Comment by s-martin** [comment]
Hi, could you add some of the info the bug template asked for?
This would help a lot. Thanks!

**Comment by TrustCompact** [comment]
> Hi, could you add some of the info the bug template asked for?
> This would help a lot. Thanks!

Totally sorry for the missing details. I put them into the OP, but forgot to remove the commenting-tags and therefore these details were not shown :( Fixed this.

If further information is needed I would be glad to post it!

**Comment by s-martin** [comment]
What’s the output of `journalctl -u lighttpd`?

Did you try to reboot?

And what’s the output of `journalctl -u mpd`?

**Comment by TrustCompact** [comment]
> What’s the output of `journalctl -u lighttpd`?

```
journalctl -u lighttpd
-- Logs begin at Tue 2020-04-14 20:17:01 CEST, end at Tue 2020-04-14 20:30:08 CEST. --
Apr 14 20:17:10 Phoniebox systemd[1]: Starting Lighttpd Daemon...
Apr 14 20:17:12 Phoniebox systemd[1]: Started Lighttpd Daemon.
Apr 14 20:17:26 Phoniebox sudo[959]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:27 Phoniebox sudo[970]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                           close
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:36 Phoniebox sudo[981]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:36 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:37 Phoniebox sudo[992]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session closed for user root
lines 1-23...skipping...
-- Logs begin at Tue 2020-04-14 20:17:01 CEST, end at Tue 2020-04-14 20:30:08 CEST. --
Apr 14 20:17:10 Phoniebox systemd[1]: Starting Lighttpd Daemon...
Apr 14 20:17:12 Phoniebox systemd[1]: Started Lighttpd Daemon.
Apr 14 20:17:26 Phoniebox sudo[959]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:27 Phoniebox sudo[970]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                           close
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:36 Phoniebox sudo[981]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:36 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:37 Phoniebox sudo[992]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:44 Phoniebox sudo[1005]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:44 Phoniebox sudo[1005]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:44 Phoniebox sudo[1005]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:44 Phoniebox sudo[1016]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                            close
Apr 14 20:28:44 Phoniebox sudo[1016]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:44 Phoniebox sudo[1016]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:50 Phoniebox sudo[1027]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:50 Phoniebox sudo[1027]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:50 Phoniebox sudo[1027]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:56 Phoniebox sudo[1039]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:56 Phoniebox sudo[1039]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:56 Phoniebox sudo[1039]: pam_unix(sudo:session): session closed for user root
Apr 14 20:29:03 Phoniebox sudo[1050]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:29:03 Phoniebox sudo[1050]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:29:03 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:29:03 Phoniebox sudo[1050]: pam_unix(sudo:session): session closed for user root
lines 1-48
-- Logs begin at Tue 2020-04-14 20:17:01 CEST, end at Tue 2020-04-14 20:30:08 CEST. --
Apr 14 20:17:10 Phoniebox systemd[1]: Starting Lighttpd Daemon...
Apr 14 20:17:12 Phoniebox systemd[1]: Started Lighttpd Daemon.
Apr 14 20:17:26 Phoniebox sudo[959]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:27 Phoniebox sudo[970]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                           close
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:36 Phoniebox sudo[981]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:36 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:37 Phoniebox sudo[992]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:44 Phoniebox sudo[1005]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:44 Phoniebox sudo[1005]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:44 Phoniebox sudo[1005]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:44 Phoniebox sudo[1016]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                            close
Apr 14 20:28:44 Phoniebox sudo[1016]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:44 Phoniebox sudo[1016]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:50 Phoniebox sudo[1027]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:50 Phoniebox sudo[1027]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:50 Phoniebox sudo[1027]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:56 Phoniebox sudo[1039]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:56 Phoniebox sudo[1039]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:56 Phoniebox sudo[1039]: pam_unix(sudo:session): session closed for user root
Apr 14 20:29:03 Phoniebox sudo[1050]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:29:03 Phoniebox sudo[1050]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:29:03 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:29:03 Phoniebox sudo[1050]: pam_unix(sudo:session): session closed for user root
lines 1-48
-- Logs begin at Tue 2020-04-14 20:17:01 CEST, end at Tue 2020-04-14 20:30:08 CEST. --
Apr 14 20:17:10 Phoniebox systemd[1]: Starting Lighttpd Daemon...
Apr 14 20:17:12 Phoniebox systemd[1]: Started Lighttpd Daemon.
Apr 14 20:17:26 Phoniebox sudo[959]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:26 Phoniebox sudo[959]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:27 Phoniebox sudo[970]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                           close
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:27 Phoniebox sudo[970]: pam_unix(sudo:session): session closed for user root
Apr 14 20:17:36 Phoniebox sudo[981]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:17:36 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:17:36 Phoniebox sudo[981]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:37 Phoniebox sudo[992]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                           currentsong
                                           close
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:37 Phoniebox sudo[992]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:44 Phoniebox sudo[1005]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:44 Phoniebox sudo[1005]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:44 Phoniebox sudo[1005]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:44 Phoniebox sudo[1016]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo playlistinfo
                                            close
Apr 14 20:28:44 Phoniebox sudo[1016]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:44 Phoniebox sudo[1016]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:50 Phoniebox sudo[1027]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:50 Phoniebox sudo[1027]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:50 Phoniebox sudo[1027]: pam_unix(sudo:session): session closed for user root
Apr 14 20:28:56 Phoniebox sudo[1039]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:28:56 Phoniebox sudo[1039]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:28:56 Phoniebox sudo[1039]: pam_unix(sudo:session): session closed for user root
Apr 14 20:29:03 Phoniebox sudo[1050]: www-data : TTY=unknown ; PWD=/home/pi/RPi-Jukebox-RFID/htdocs/api ; USER=root ; COMMAND=/bin/echo status
                                            currentsong
                                            close
Apr 14 20:29:03 Phoniebox sudo[1050]: pam_unix(sudo:session): session opened for user root by (uid=0)
Apr 14 20:29:03 Phoniebox lighttpd[904]: echo: write error: Broken pipe
Apr 14 20:29:03 Phoniebox sudo[1050]: pam_unix(sudo:session): session closed for user root
```


> Did you try to reboot?

Sure - several times. No difference.
It even makes no difference from which machine I open the web interface. Mobile, Desktop, Notebook - also tried different browsers.
 
> And what’s the output of `journalctl -u mpd`?

```
journalctl -u mpd
-- Logs begin at Tue 2020-04-14 20:17:01 CEST, end at Tue 2020-04-14 20:31:29 CEST. --
Apr 14 20:17:10 Phoniebox systemd[1]: Starting Music Player Daemon...
Apr 14 20:28:40 Phoniebox systemd[1]: Started Music Player Daemon.
```

**Comment by TrustCompact** [comment]
Well, I found the problem.
I had mounted two network folders from OMV via etc/fstab to avoid having file duplicates.
Removing these mounted folders gets the normal speed of the web interface back.

Anyway it would be nice to be able to use mounted folders without having these preformance problems. Access to the mounted folders via console or file explorer works in normal speed.

Or are there any other ways to achieve using network files?

Thanks!
